airbyte/manifest.json

```diff
@@ -31,7 +31,10 @@
       },
       "service_checks": {
         "metadata_path": "assets/service_checks.json"
-      }
+      },
+      "process_signatures": [
+        "airbyte/bin"
```

Comment:
Airbyte starts several services with binaries in this path. Matching only for it leaves us flexible, we're not tied to one particular service being up, in case there's some custom deployment of airbyte.